testing/spectest/shared/common/light_client/update_ranking.go

```diff
@@ -20,6 +21,21 @@ import (
 // RunLightClientUpdateRankingTests executes "light_client/update_ranking/pyspec_tests/update_ranking" tests.
 func RunLightClientUpdateRankingTests(t *testing.T, config string, v int) {
 	require.NoError(t, utils.SetConfig(t, config))
+	if v >= 1 {
+		params.BeaconConfig().AltairForkEpoch = 0
+	}
+	if v >= 2 {
+		params.BeaconConfig().BellatrixForkEpoch = 0
+	}
+	if v >= 3 {
+		params.BeaconConfig().CapellaForkEpoch = 0
+	}
+	if v >= 4 {
+		params.BeaconConfig().DenebForkEpoch = 0
+	}
+	if v >= 5 {
+		params.BeaconConfig().ElectraForkEpoch = 0
+	}
```

Comment:
I know this is test code, but we can avoid these "magic numbers" by using the version enum.

```suggestion
	if v >= version.Altair {
		params.BeaconConfig().AltairForkEpoch = 0
	}
	if v >= version.Bellatrix {
		params.BeaconConfig().BellatrixForkEpoch = 0
	}
	if v >= version.Capella {
		params.BeaconConfig().CapellaForkEpoch = 0
	}
	if v >= version.Deneb {
		params.BeaconConfig().DenebForkEpoch = 0
	}
	if v >= version.Electra {
		params.BeaconConfig().ElectraForkEpoch = 0
	}
```
Do you want to handle Fulu while you are here? It's coming soon...

Reply:
thanks. will do.